storage/src/robocorp/storage/_requests.py

```diff
@@ -0,0 +1,180 @@
+import json
```

Comment:
Next PR: This will be used from a common single place instead of being duplicated.